framework/src/controller/bus.ts

```diff
@@ -253,25 +268,27 @@ export class Bus {
 			}
 		}
 
-		// For child process channel
-		return new Promise((resolve, reject) => {
-			(channelInfo.rpcClient as RPCClient).call(
-				'invoke',
-				action.toJSONRPCRequest(),
-				(err: Error | undefined, data: JSONRPC.ResponseObjectWithResult<T>) => {
-					if (err) {
-						return reject(parseError(action.id, err));
-					}
-
-					return resolve(data);
-				},
-			);
+		return new Promise(resolve => {
+			this._rpcRequestIds.add(action.id as string);
```

Comment:
Because the `action.id` would always be a string, can we adjust it type and remove this `as string` from many places. May be another issue for it?

Reply:
Will be resolved in https://github.com/liskhq/lisk-sdk/issues/6615